snapcraft/tests/unit/store/test_store_client.py

```diff
@@ -1506,3 +1517,13 @@ def test_conflicting_force(self):
             # force the update, even on conflicts!
             result = self.client.push_binary_metadata('basic', metadata, True)
         self.assertIsNone(result)
+
+    def test_snap_not_found(self):
```

Comment:
It seems you have the same test twice, here. Is that true, or am I missing something?

Reply:
Ah, binary metadata, very good!

Reply:
Hi sorry I made a mistake it should be `push_binary_metadata` called inside this function, to cover this line too. [https://github.com/snapcore/snapcraft/blob/master/snapcraft/storeapi/_store_client.py#L302](https://github.com/snapcore/snapcraft/blob/master/snapcraft/storeapi/_store_client.py#L302)

Reply:
Yeah I saw that, no problem :) .